homeassistant/components/iaqualink/config_flow.py

```diff
@@ -0,0 +1,59 @@
+"""Config flow to configure zone component."""
+
+from typing import Optional
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers import ConfigType
+
+from .const import DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class AqualinkFlowHandler(config_entries.ConfigFlow):
+    """Aqualink config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    async def async_step_user(self, user_input: Optional[ConfigType] = None):
+        """Handle a flow start."""
+        from iaqualink import AqualinkClient
+
+        errors = {}
+
+        if user_input is not None:
+            # Supporting a single account.
+            entries = self.hass.config_entries.async_entries(DOMAIN)
+            if entries:
+                return self.async_abort(reason="already_setup")
+
+            username = user_input[CONF_USERNAME]
+            password = user_input[CONF_PASSWORD]
+
+            try:
+                aqualink = AqualinkClient(username, password)
+                await aqualink.login()
+                return await self.async_step_import(user_input)
+            except Exception:  # pylint: disable=W0703
```

Comment:
Please only except expected exceptions.

Reply:
The underlying API currently uses Exception.

Reply:
Please update the library to raise library specific exceptions. Make one base exception, specific to the library, that inherits from `Exception` and then further child exceptions based on the first exception as needed.

Thrn we can import those exceptions here as needed use when excepting errors.

Reply:
Updated the library, will reflect in the next update.